apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -733,18 +738,23 @@ export const renderError = ({
   withDescription?: boolean;
   Icon?: (props: { color?: string | undefined; size?: number | undefined }) => JSX.Element;
 }) => {
+  let tmpError = error;
```

Comment:
I think it's about the line 752 to update the error type to a specific type. But you can do this after the first 3 assertions, it will be more clear. If the first 3 catches do not catch the error, then you do a check (`error instanceof CompleteExchangeError && error.message === 'User refused'`) to see whether the error type needs an update.

Reply:
I believe this is correct, if we aren't doing any variable assignment at line `752`, i can removed the `let tmpError`